bundle/manifest/manifest.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+// Package manifest implements helper functions for the stored manifest.
+package manifest
+
+import (
+	"context"
+	"fmt"
+	"github.com/open-policy-agent/opa/bundle"
+	"github.com/open-policy-agent/opa/storage"
+	"github.com/open-policy-agent/opa/util"
+)
+
+var bundlesBasePath = storage.MustParsePath("/system/bundles")
```

Comment:
Is it possible to continue using the old path if the deprecated `bundle` is in use? The reason is that there are a few places where we rely on this path today (opa-istio-plugin, example-api-authz-go). We also depend on that in the file loader as well. When you `opa run` the runtime configures the loader to include the manifest metadata into the data document (and the loader sticks it into /system/bundle/...) This lets people load a bundle via the command line and then get the revision in the provenance API, decision logs, etc.

Reply:
Yep. The latest version will always write to the old path from bundle downloads and anything loaded via the file loader. The bundle download path will always write to both locations and everywhere in our code base we only read from the new one.

What this means is that file loaded bundles are not initially supported with multi bundle configured opa's. In practice we have very similar limitations today with file loaded bundles combined with the single bundle service config... at least now we know about it. I intend to give a better CLI/file path loaded solution with the new `-b` flag that should help give a migration path.